glotaran/deprecation/deprecation_utils.py

```diff
@@ -607,6 +609,11 @@ def deprecate_module_attribute(
         ``glotaran.parameter.ParameterGroup``
     to_be_removed_in_version : str
         Version the support for this usage will be removed.
+    module_load_overwrite : str
+        Overwrite the location the functionality will be set from.
+        This allows preserving functionality without polluting new
```

Comment:
This function could do with some cleaning up in terms of documentation, @s-weigand.